frontend/src/components/Login/Input.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Input.css';
+
+interface InputProps {
+  name: string;
+  type: 'string' | 'email' | 'password' | 'number';
+  placeholder: string;
+  onChange: (e: React.FormEvent<HTMLInputElement>) => void;
+}
+
+export function Input({ name, type, placeholder, onChange }: InputProps) {
+  return (
+    <div className="input-box">
```

Comment:
This `div` is not needed. You can move the styles applied to this in the `input-field` selector and remove the `div` and its CSS class.